int8 A
---
int8 result